game-core/src/main/java/games/strategy/engine/auto/update/EngineVersionCheck.java

```diff
@@ -40,6 +49,16 @@ static void checkForLatestEngineVersionOut() {
     }
   }
 
+  private static Properties getProperties() {
+    final Properties props = new Properties();
+    try {
+      props.load(new URL(TRIPLEA_VERSION_LINK).openStream());
```

Comment:
Is the url stream ever closed?

Reply:
Looks like not, good spot! This piece of code will be replaced pretty soon, let's keep an eye out for the fix in the next few iterations.